### ISSUE TITLE
Takes multiple tries to left-click on active tabs

### ISSUE THREAD

**Issue by Arthaey** [Lej77/move-unloaded-tabs-for-tree-style-tab#2]
It seems like it's "harder" to left-click on active/loaded tabs since I've installed this addon. I often have to click twice, or very carefully click the background of the tab but not the tab's title text.

I'm guessing it has something to do with click targets or maybe event bubbling/propogation? But I haven't looked at your addon's code, so maybe not. :)

**Comment by Lej77** [comment]
I can't say I have noticed something like that but I also can't say its impossible. This extension uses [Tree Style Tab's API](https://github.com/piroor/treestyletab/wiki/API-for-other-addons#when-a-tab-is-clicked) to prevent a clicked tab from becoming active. It doesn't interact directly with Tree Style Tab's sidebar's DOM. Still, event bubbling might affect what events are sent by Tree Style Tab and how Tree Style Tab processes the result from the events.

Anyway I did some testing and it seems like the latest versions of Tree Style Tab doesn't activate tabs when they are moved (drag and dropped) anymore. Therefore most of the functionality provided by this extension isn't really needed anymore and moving unloaded tabs without loading them should be possible with just Tree Style Tab installed. One functionality that this extension can still provide is to prevent long pressed tabs from becoming active but that is a minor thing and isn't even enabled with the default settings.
